src/main/java/com/ericsson/gerrit/plugins/eiffel/messaging/EiffelEventSender.java

```diff
@@ -60,16 +60,19 @@ public EiffelEventSender(EiffelPluginConfiguration pluginConfig, HttpRequest htt
     }
 
     /**
-     * Sends a REMReM eiffel message using
+     * Sends a REMReM Eiffel message to the generateAndPublish endpoint. RuntimeException is thrown
+     * when a HttpRequestFailedException occurs so that the retry logic works.
```

Comment:
I don't get the comment. I guess `HttpRequestFailedException` triggers the retry logic so that the reason why we need to throw it?

Reply:
Ah yes, this has been changed, HttpRequestFailedException is of the type RuntimeException now. I must update the docs.